app/javascript/listings/singleListing.jsx

```diff
@@ -82,7 +92,18 @@ export const SingleListing = ({
 };
 
 SingleListing.propTypes = {
-  listing: PropTypes.object.isRequired,
+  listing: PropTypes.shape({
+    id: PropTypes.number,
+    category: PropTypes.string,
+    contact_via_connect: PropTypes.bool,
+    location: PropTypes.string,
+    processed_html: PropTypes.string,
+    slug: PropTypes.string,
+    title: PropTypes.string,
+    user_id: PropTypes.number,
+    tag_lis: PropTypes.array,
```

Comment:
there's a typo here, it should be `tag_list`, also possibly `arrayOf(string)`

Reply:
@rhymes thanks for the catch